Resources/Storage/GFAL2_XROOTStorage.py

```diff
@@ -52,11 +51,14 @@ def __init__( self, storageName, parameters ):
     self.protocolParameters['WSUrl'] = 0
     self.protocolParameters['SpaceToken'] = 0
 
-
-
     # We don't need extended attributes for metadata
     self._defaultExtendedAttributes = None
 
+    # Because some storages are configured to use krb5 auth first
+    # we end up in trouble for interactive sessions. This
+    # environment variable enforces the use of certificates
+    if 'XrdSecPROTOCOL' not in os.environ:
+      os.environ['XrdSecPROTOCOL'] = 'gsi'
```

Comment:
This needs to be "gsi,unix" for third party copy to work
See this ticket for explanations: https://cern.service-now.com/service-portal/view-incident.do?n=INC1168179